plugin.rb

```diff
@@ -208,15 +211,15 @@ def after_authenticate(auth_token)
       campaign[:attributes][:published_at].present?
     end
 
-    unless published_campaign
-      result.failed = true
-      result.failed_reason = "This forum is for launched Patreon creators only. Visit <a href='https://patreon.com/faq'>patreon.com/faq</a> if you need further help. Thanks!".html_safe
-    else
+    # This will allow all @patreon.com email addresses and the *patron1@mutations.ltd (for testing)
+    bypass_creator_logic = /.*(@patreon\.com|patron1@mutations\.ltd)$/.match?(result.email)
```

Comment:
A few comments:

* I'd recommend moving the pattern to a frozen constant
* You might consider sanitizing/normalizing/validating the user input before passing to the regexp
* This might make a good method _(e.g. `should_bypass_creator_logic?(result.email)`)_

Reply:
OK I will do that.

Reply:
@phlipper Thanks. I fixed it up and I think it is much cleaner now,  I didn't add a sanitize check on the email in this PR, I have more to do on this logic in a different story and Mindy wants to allow staff to login, which is what this PR allows.